src/redux/__test__/reducers.test.js

```diff
@@ -0,0 +1,22 @@
+import coinReducer, { fetchCryptos } from '../crypto/crypto';
+// import fetchCryptos from '../crypto/crypto';
```

Comment:
- [ ] Kindly remove commented code from your file. Also, I suggest you put all tests in one file. I can see you are putting them separately. But, it would be sorted if you place them in one place.

Reply:
Well resolved ✔️